src/app/modules/submission/submission.errors.ts

```diff
@@ -241,3 +241,27 @@ export class ExpectedResponseNotFoundError extends ApplicationError {
     super(message, undefined, ErrorCodes.SUBMISSION_EXPECTED_RESPONSE_NOT_FOUND)
   }
 }
+
+export class MrfReminderInvalidWorkflowStepError extends ApplicationError {
+  constructor(
+    message = 'Unable to send reminder as no valid pending workflow step is found.',
+  ) {
+    super(
+      message,
+      undefined,
+      ErrorCodes.SUBMISSION_MRF_REMINDER_INVALID_PENDING_STEP,
+    )
+  }
+}
+
+export class MrfReminderRecipientEmailsEmptyError extends ApplicationError {
+  constructor(
+    message = 'Unable to send reminder as no recipient emails found for the pending step.',
```

Comment:
Also on the same note, to attempt to make it less technical.

```suggestion
    message = 'Unable to send reminder as no recipient emails found.',
```